mirrord/config/src/config.rs

```diff
@@ -17,8 +17,8 @@ pub enum ConfigError {
     #[error("value for {1:?} not provided in {0:?} (env override {2:?})")]
     ValueNotProvided(&'static str, &'static str, Option<&'static str>),
 
-    #[error("value {0:?} for {1:?} is invalid.")]
-    InvalidValue(String, &'static str),
+    #[error("invalid {1} value `{0}`: {2}")]
```

Comment:
Probably worth naming the arguments or adding a comment to explain what you expect the error to be called with
```suggestion
    // Call with: value we got from user, field/envvar name, error message
    #[error("invalid {1} value `{0}`: {2}")]
```

Reply:
Changed to named fields ^^